tests/test_youtube_thumbnail.php

```
@@ -0,0 +1,50 @@
+<?php
+/* Tests for getYoutubeThumbnail() — used in caBuildRepoMediaSection / popup
+   video tile rendering. Pure URL→URL transform. */
+
+require_once __DIR__ . "/lib.php";
+require_once "/usr/local/emhttp/plugins/community.applications/include/helpers.php";
```

PHPStan (2.1.54)

[error] Path in require_once() "/usr/local/emhttp/plugins/community.applications/include/helpers.php" is not a file or it does not exist.


(requireOnce.fileNotFound)